src/GovUk.Education.ExploreEducationStatistics.Content.Model/Database/ApplicationDbContext.cs

```diff
@@ -811,7 +812,7 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
                   Summary = "",
                   TopicId = new Guid("1a9636e4-29d5-4c90-8c07-f41db8dd019c"),
                   Slug = "secondary-and-primary-schools-applications-and-offers",
-                  LegacyPublicationUrl = new Uri("https://www.gov.uk/government/collections/statistics-school-applications#documents")
+                  // LegacyPublicationUrl = new Uri("https://www.gov.uk/government/collections/statistics-school-applications#documents")
```

Comment:
I'd probably remove this line if its no longer needed

Reply:
done